app/assets/stylesheets/components/_session_form.scss

```diff
@@ -0,0 +1,39 @@
+.session_form{
+  align-items: center;
+  display: flex;
+  font-weight: 600;
+  height: 100%;
+  justify-content: center;
+  width: 100%;
+
+  &__card{
+    background-color: white;
```

Comment:
Use variables to add color, can made a specific file for it, could be name it variables.scss